MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -448,10 +460,11 @@ KnownImages GetPackageImageId (FeedKind kind)
 			}
 		}
 
-		CompletionItem CreateNuGetVersionCompletionItem (string version, FeedKind kind)
+		CompletionItem CreateNuGetCompletionItem (string info, FeedKind kind, XmlCompletionItemKind xmlCompletionItemKind)
 		{
 			var kindImage = DisplayElementFactory.GetImageElement (GetPackageImageId (kind));
-			var item = new CompletionItem (version, this, kindImage);
+			var item = new CompletionItem (info, this, kindImage);
+			item.Properties.AddProperty (typeof (XmlCompletionItemKind), xmlCompletionItemKind);
```

Comment:
FWIW you can use the AddKind extension method here

Reply:
Using `item.AddKind (xmlCompletionItemKind)` now